app/src/main/res/values/colors.xml

```diff
@@ -27,4 +27,9 @@
     <color name="color_yellow_dark">#FFAA16</color>
     <color name="color_orange_light_white">#FFB185</color>
     <color name="color_orange_light_transparent">#FFE3D7</color>
+    <color name="color_white_transparent_light">#BFE2E2E2</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.color_white_transparent_light` appears to be unused
 Unused resources make applications larger and slow down builds.